File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/imprimir_cupom_sat_cancelamento.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class ImprimirCupomSatCancelamento extends BridgeCommand {
  final String _xml;
  final String _assQRCode;

  ImprimirCupomSatCancelamento(this._xml, this._assQRCode)
      : super('ImprimirCupomSatCancelamento');

  @override
  functionParameters() {
    return '"xml"'
        ':'
        '"'
        '$_xml'
        '"'
        ','
        '"assQRCode"'
        ':'
        '"$_assQRCode"';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/Commands/ImprimirCupomSatCancelamento.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class ImprimirCupomSatCancelamento(private val xml: String, private val assQRCode: String) :
    BridgeCommand("ImprimirCupomSatCancelamento") {
    override fun functionParameters(): String {
        return "\"xml\"" + ":" + "\"" + xml + "\"" + "," +
                "\"assQRCode\"" + ":" + "\"" + assQRCode + "\""
    }
}